MixItUp.WPF/Controls/Settings/AdvancedSettingsControl.xaml.cs

```diff
@@ -80,6 +80,54 @@ private async void RestoreSettingsButton_Click(object sender, RoutedEventArgs e)
                 string filePath = ChannelSession.Services.FileService.ShowOpenFileDialog("Mix It Up Settings (*.mixitup)|*.mixitup|All files (*.*)|*.*");
                 if (!string.IsNullOrEmpty(filePath))
                 {
```

Comment:
The code below extracts the file to a temp dir, finds the XML file, pulls the version and shows errors as appropriate.